server/src/test/java/org/elasticsearch/action/search/TransportSearchActionTests.java

```diff
@@ -256,100 +257,117 @@ public void testProcessRemoteShards() {
         ) {
             RemoteClusterService service = transportService.getRemoteClusterService();
             assertFalse(service.isCrossClusterSearchEnabled());
```

Comment:
Should we remove this line? It seems unrelated to what we are testing and is confusing to the reader.

Reply:
++, I removed the TransportServices from tests: https://github.com/elastic/elasticsearch/pull/95894/commits/428e098e37a4f81abbcf486912b6bbd4ebc38580